packages/api/src/libs/queries.ts

```diff
@@ -1,13 +1,51 @@
+import assert from "assert";
 // allow for more complex queries, joins and shared queries between services
-import type { AppState, CurrencySymbol } from "..";
-import uma from "@uma/sdk";
+import type { AppState, CurrencySymbol, PriceSample } from "..";
+import * as uma from "@uma/sdk";
 import { calcGcr } from "./utils";
-import Promise from "bluebird";
+import bluebird from "bluebird";
```

Comment:
typescript doesnt like when this is named "Promise", i've found there are type incompatibilities